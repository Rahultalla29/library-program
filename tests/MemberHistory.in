MEMBER HISTORY
ADD COLLECTION sample1.csv
ADD MEMBER Naruto Uzumaki
ADD MEMBER Kakashi Hatake
MEMBER HISTORY 100000
MEMBER HISTORY 100001
MEMBER HISTORY 100002
RENT 100000 111111
RENT 100000 111112
RENT 100000 111113
RENT 100000 111114
RENT 100000 111115
RENT 100000 111116
RENT 100000 111117
RELINQUISH 100000 111111
MEMBER HISTORY 100000
RELINQUISH ALL 100000
MEMBER HISTORY 100000
EXIT
