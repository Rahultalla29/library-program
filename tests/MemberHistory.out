user: No members in system.

user: 17 books successfully added.

user: Success.

user: Success.

user: No rental history for member.

user: No rental history for member.

user: No such member in system.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)

user: Ending Library process.
